### PR TITLE
[Dark mode] Enhancements

### PR DESCRIPTION
**Pull request by ronronscelestes**
## What

- Synced specify for new color token values
- Synced specify for new color tokens (buttonPrimary600 / buttonPrimary500)
- Synced specify for new dark mode shadows
- Used buttonPrimary600 for all button background components variant default/primary
- Used buttonPrimary500 for all button hover components variant default/primary
- Added border to the Table component
- Uncommented dark mode tests that used to fail because of button components
- Changed InitialWrapper background used in MainNav

### CODE REVIEW

**Review by maevalienard — APPROVED** [review]
LGTM 🔥 Thank you!

File: .specifyrc.json [hunk 1]
```diff
@@ -128,12 +128,79 @@
       ]
     },
     {
-      "name": "Design Tokens / shadows",
-      "path": "packages/strapi-design-system/src/themes/shadows.js",
+      "name": "Design Tokens / light shadows",
```
Comment (HichamELBSI) [inline .specifyrc.json:131]: 🔥

File: packages/strapi-design-system/src/Avatar/Avatar.js [hunk 2]
```diff
@@ -62,33 +62,22 @@ export const Avatar = ({ src, alt, preview }) => {
...
 `;
 
 export const Initials = ({ children }) => {
   return (
-    <InitialsWrapper justifyContent="center">
-      <Typography fontWeight="bold" textColor="neutral0" fontSize={0} textTransform="uppercase">
+    <InitialsWrapper
+      borderRadius="50%"
+      width={`${avatarSize}px`}
+      height={`${avatarSize}px`}
```
Comment (HichamELBSI) [inline packages/strapi-design-system/src/Avatar/Avatar.js:76]: I'm wondering if we should turn this sizes into rem units so it can scale

Comment (HichamELBSI) [reply packages/strapi-design-system/src/Avatar/Avatar.js:76]: wdyt?

File: packages/strapi-design-system/src/Button/__tests__/Button.e2e.js [hunk 3]
```diff
@@ -45,46 +45,45 @@ test.describe.parallel('Button', () => {
...
-  //   });
-  // });
+    test.describe('disabled', () => {
+      test('triggers axe on the document', async ({ page }) => {
+        await page.goto('/iframe.html?id=design-system-components-button--disabled&viewMode=story&theme=dark');
+        await injectAxe(page);
+        await checkA11y(page);
+      });
+    });
+  });
```
Comment (HichamELBSI) [inline packages/strapi-design-system/src/Button/__tests__/Button.e2e.js:88]: awesome

File: packages/strapi-design-system/src/Button/utils.js [hunk 4]
```diff
@@ -48,6 +48,13 @@ export const getHoverStyle = ({ theme, variant }) => {
     `;
   }
 
+  if (variant === 'default') {
```
Comment (HichamELBSI) [inline packages/strapi-design-system/src/Button/utils.js:51]: I think we'll have to turn these strings into constants at some points.

File: packages/strapi-design-system/src/FocusTrap/__tests__/FocusTrap.e2e.js [hunk 5]
```diff
@@ -115,13 +115,12 @@ test.describe.parallel('FocusTrap', () => {
...
-  //   });
-  // });
+  test.describe('dark mode', () => {
+    test('triggers axe on the document', async ({ page }) => {
+      // This is the URL of the Storybook Iframe
+      await page.goto('/iframe.html?id=design-system-technical-components-focustrap--base&viewMode=story&theme=dark');
+      await injectAxe(page);
+      await checkA11y(page);
+    });
+  });
```
Comment (HichamELBSI) [inline packages/strapi-design-system/src/FocusTrap/__tests__/FocusTrap.e2e.js:125]: Great improvements!

File: packages/strapi-design-system/src/Layout/__tests__/HeaderLayout.e2e.js [hunk 6]
```diff
@@ -71,14 +71,13 @@ test.describe.parallel('HeaderLayout', () => {
...
-  // });
+  test.describe('dark mode', () => {
+    test.describe('base', () => {
+      test('triggers axe on the document', async ({ page }) => {
+        await page.goto('/iframe.html?id=design-system-components-headerlayout--base&viewMode=story&theme=dark');
+        await injectAxe(page);
+        await checkA11y(page);
+      });
+    });
+  });
```
Comment (HichamELBSI) [inline packages/strapi-design-system/src/Layout/__tests__/HeaderLayout.e2e.js:82]: Great!

File: packages/strapi-design-system/src/Avatar/Avatar.js (hunk 2 above)
Comment (ronronscelestes) [reply packages/strapi-design-system/src/Avatar/Avatar.js:76]: sounds good indeed, on it!

File: packages/strapi-design-system/src/Button/utils.js (hunk 4 above)
Comment (ronronscelestes) [reply packages/strapi-design-system/src/Button/utils.js:51]: I could try by adding a constants file in utils?

File: packages/strapi-design-system/src/Avatar/Avatar.js (hunk 2 above)
Comment (ronronscelestes) [reply packages/strapi-design-system/src/Avatar/Avatar.js:76]: mh for some of them it seems to break things

if I do 
```
<AvatarImg src={src} alt={alt} width={`${avatarSize / 16}rem`} height={`${avatarSize / 16}rem`} />
``` 
it gives me this (yes the little dot it the AvatarImg) 😅
image

and for this one
```
        <PreviewContainer
          aria-hidden
          alt=""
          width={`${previewSize / 16}rem`}
          height={`${previewSize / 16}rem`}
          src={preview === true ? src : preview}
        />
```
I get (little dot is the preview:
image

Comment (ronronscelestes) [reply packages/strapi-design-system/src/Avatar/Avatar.js:76]: I can update AvatarImgWrapper and the Overlay without breaking things though

Comment (HichamELBSI) [reply packages/strapi-design-system/src/Avatar/Avatar.js:76]: Wut? 😂  you can rollback to px if you can't fix it for the moment 👍🏽

**Review by HichamELBSI — APPROVED** [review]
Awesome thanks! Good job

**Pull request merged**